content/influxdb/cloud/write-data/troubleshoot.md

```diff
@@ -46,28 +47,25 @@ To resolve partial writes and rejected points, see [troubleshoot failures](#trou
 ## Review HTTP status codes
 
 InfluxDB uses conventional HTTP status codes to indicate the success or failure of a request.
+
+{{% warn %}}
+#### Asynchronous writes
+
+`204` doesn't indicate a successful write operation given writes are asynchronous. If some of your data did not write to the bucket, see how to [check for rejected points](#review-rejected-points).
```

Comment:
```suggestion
`204` indicates InfluxDB validated the request data format. Because data is written to InfluxDB asynchronously, data may not yet be written to a bucket. If some of your data did not write to the bucket, see how to [check for rejected points](#review-rejected-points).
```

Reply:
Curious (maybe a question for engineering), is there a way that we could be more explicit? Something along the lines of exactly how often data is written to a bucket and/or when folks should check for unwritten data?